index.html

```diff
@@ -666,6 +666,43 @@
   </div>
 </div>
 
+<div class="row location section">
+  <div class="col-xs-10 col-xs-offset-1 col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1">
+    <div class="row">
+      <div class="col-sm-12">
+        <h3>Location</h3>
+        <div class="col-sm-3 col-xs-12">
+          <img src="images/location-berlin.png" alt="Image of location" class="img-circle img-responsive center-block">
```

Comment:
We shouldn't use PNG for image - JPGs will have a much smaller file size at the same quality.

Also all the other images should be removed from this PR.